cylc/flow/scripts/cylc.py

```diff
@@ -24,7 +24,7 @@
 from cylc.flow import __version__
 
 # These will be ported to python as click commands
-bash_commands = ["cylc-graph-diff", "cylc-jobscript"]
+bash_commands = ["cylc-graph-diff"]
```

Comment:
Just one left!

Reply:
Wooty! Are there plans for this one to go too?

Reply:
We might be able to merge it into `cylc graph` e.g. `cylc graph --diff`.